CMakeLists.txt

```diff
@@ -54,13 +54,15 @@ else ()
 
 	IF (CMAKE_SYSTEM_PROCESSOR MATCHES "^(i.86|x86(_64)?)$")
 		if (NANO_SIMD_OPTIMIZATIONS OR RAIBLOCKS_SIMD_OPTIMIZATIONS)
-			add_compile_options(-msse4)
+			add_compile_options(-msse4 -maes)
```

Comment:
Should there be a NANO_ENABLE_AES flag? Seems like a cpu can have SSE4 support but not the AES instruction set. The -maes use later on inside ENABLE_AVX2 is probably fine.

Reply:
But we didn't document that SIMD optimizations = SSE4.2 only

Reply:
Should we add docs that AES support is required for SIMD optimizations then? For instance, there are corei7's with sse4, but not avx/aes.

Reply:
Enabled by default for SIMD now, but can be disabled if required

Reply:
Perfect